README.md

```diff
@@ -37,6 +37,209 @@
+
+	init(name: String, height: Double, favoriteFood: String) {
+        	self.name = name
+        	self.height = height
+        	self.favoriteFood = favoriteFood
+    	}
+
+Your Person class now looks like this.
+
+	class Person {
+
+    		// here are the properties
+    		var name: String
+    		var height: Double
+    		var favoriteFood: String
+
+    		// here is the initializer method
+    		init(name: String, height: Double, favoriteFood: String) {
+        		self.name = name
+        		self.height = height
+        		self.favoriteFood = favoriteFood
+    		}
+	}
+
+You can create an instance of this class and assign it to a local variable using this code;
+
+		let jim = Person(name: "Jim", height: 64.0, favoriteFood: "Pasta")
+
+## Create a Text Description Computed Variable for your Person Class
+How do you know that this works? How can you see what you’ve created? One way is to add a new property to the class.  Call it `textDescription` (because that’s what it will be). Here’s how you create that variable in your Class definition:
+
+		var textDescription: String {
+			return "Name: \(self.name) \nHeight: \(self.height) \nFavorite Food: \(self.favoriteFood)"
+		}
+ 
+## Create an Instance of the Person Class
+Now, you can create an instance of the class, and it will have the `textDescription` variable created automatically from the values you supply for the class. Here’s how you create an instance of the class, and then print `textDescription`.
+
+		let jim = Person(name: "Jim", height: 6.0, favoriteFood: “Pasta")
+		print (jim.description)
+
+Here’s the result
+
+		// Name: Jim Height: 6.0 Favorite Food: Pasta
+
+## Generalizing the Person Class
+You can use this as a model for all sorts of other classes: Dog, Cat, Building, Car, or anything you want. Just declare the class, declare the relevant variables, build the initializer, and create a `textDescription` variable that you lets you print out the data inside the class.
+
+Of course, the actual properties inside these other classes will vary. For example, if you create a Building class, here’s what you might need. Instead of name, you might need address. Height will be ok — both people and buildings have height. Buildings don’t have favorite foods, so you might just omit that. Here’s what you get when you create a Building class with address and height:
+
+		class Building {
+
+    			// here are the properties
+    			var address: String
+    			var height: Double
+			var textDescription: String {
+        			return "Address: \(self.address) \nHeight: \(self.height)"
+    			}
+
+    			// here is the initializer method
+    			init(address: String, height: Double) {
+        			address = name
+        			self.height = height
+   			}
+		}
+		
+Here is an example of instantiating an instance of Building (a library in this case).
+
+		let library = Building(address: “Main Street", height: 20.0, favoriteFood: "Pasta")
+
+		print(description)
```

Comment:
Similar to the above print function call. This should be printing textDescription as follows:

```swift
print(library.textDescription)
// Address: Main Street
// Height: 20.0
```